sdk/dotnet/ApiExtensions/V1/Inputs/CustomResourceDefinitionSpecArgs.cs

```diff
@@ -0,0 +1,64 @@
+// *** WARNING: this file was generated by pulumigen. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.Kubernetes.Types.Inputs.ApiExtensions.V1
+{
+
+    /// <summary>
+    /// CustomResourceDefinitionSpec describes how a user wants their resource to appear
+    /// </summary>
+    public class CustomResourceDefinitionSpecArgs : Pulumi.ResourceArgs
+    {
+        /// <summary>
+        /// conversion defines conversion settings for the CRD.
+        /// </summary>
+        [Input("conversion")]
+        public Input<Pulumi.Kubernetes.Types.Inputs.ApiExtensions.V1.CustomResourceConversionArgs>? Conversion { get; set; }
+
+        /// <summary>
+        /// group is the API group of the defined custom resource. The custom resources are served under `/apis/&lt;group&gt;/...`. Must match the name of the CustomResourceDefinition (in the form `&lt;names.plural&gt;.&lt;group&gt;`).
```

Comment:
Are the `&lt;` and `&gt;` symbols correct?

Reply:
Yes, C# comments are XML, so `<>` should be encoded